addon/-private/system/model/internal-model.js

```diff
@@ -221,16 +220,6 @@ export default class InternalModel {
     return this.currentState.dirtyType;
   }
 
-  // DO NOT USE : purely to ease the transition in tests
-  get _attributes() {
```

Comment:
If they are already doing this *bad thing* then they have both a good thing or a slightly different *bad thing* now available to them.

Reply:
@rwjblue also, I think we are fairly sure in regard to `_attributes` (although not `_relationships`) that people ARE doing bad things. We could leave a nice assert here vs allowing it to fail with a less clear error.  Not totally sure how I feel about a deprecation for a private API on a Private (even if intimate) class. I suspect it would only encourage them to do the other bad thing (`internalModel._recordData._attributes`) instead of the good thing (implementing a custom `RecordData`).

Reply:
Ya, I’m not saying it’s blocking per-se I just wanted to call it out. It doesn’t seem directly related to the testing changes in the rest of the PR...